pkg/apis/config/v1alpha1/resmgr/policy/balloons/config.go

```diff
@@ -149,6 +149,12 @@ type BalloonDef struct {
 	// this will be the number of CPUs reserved for it even if a container
 	// would request less.
 	MinCpus int `json:"minCPUs,omitempty"`
+	// MemoryTypes lists memory types allowed to containers in a
+	// balloon. Supported types are: DRAM, HBM, PMEM. By default
+	// all memory types in the system are allowed.
+	// +kubebuilder:validation:MaxItems=3
+	// +kubebuilder:validation:items:Pattern=`^(DRAM|HBM|PMEM)$`
```

Comment:
I would also add `// +listType=set` too to avoid having duplicated values.

Reply:
Added +listType, thanks @fmuyassarov!